packages/read-model/src/Product/Detail/ProductDetailViewElasticsearchFactory.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ReadModelBundle\Product\Detail;
+
+use Shopsys\FrameworkBundle\Component\Utils\Utils;
+use Shopsys\FrameworkBundle\Model\Customer\User\CurrentCustomerUser;
+use Shopsys\FrameworkBundle\Model\Product\Product;
+use Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade;
+use Shopsys\ReadModelBundle\Brand\BrandView;
+use Shopsys\ReadModelBundle\Brand\BrandViewFactory;
+use Shopsys\ReadModelBundle\Image\ImageView;
+use Shopsys\ReadModelBundle\Image\ImageViewFacadeInterface;
+use Shopsys\ReadModelBundle\Parameter\ParameterViewFactory;
+use Shopsys\ReadModelBundle\Product\Action\ProductActionViewFactory;
+use Shopsys\ReadModelBundle\Product\PriceFactory;
+
+class ProductDetailViewElasticsearchFactory
+{
+    /**
+     * @var \Shopsys\ReadModelBundle\Image\ImageViewFacadeInterface
+     */
+    protected $imageViewFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade
+     */
+    protected $seoSettingFacade;
```

Comment:
This property is not used and I believe it should be.

Right now when SEO information is not set (by the product itself), they're not fallback to the default value (and should be)

Reply:
> This property is not used

You are right, thanks!

>Right now when SEO information is not set (by the product itself), they're not fallback to the default value (and should be)

I do not thing this should be work for factory. I would rather move this fallback right into `ProductDetailView` - what do you think?

Reply:
I don't think the view object should be dependent on any service (similar to entities)

Reply:
You do not need any extra dependency. It will be just fallback to value which is already in that class.

Reply:
I don't follow. 
In `ProductDetailViewFactory`, I see a fallback to the default value. In `ProductDetailViewElasticsearchFactory` the value is obtained from elasticsearch (where the fallback value is not stored).

Reply:
Solved by adding fallback into `ProductDetailViewElasticsearchFactory`